classic/form/field/GeocoderComboBox.js

```diff
@@ -207,7 +207,7 @@ Ext.define('GeoExt.form.field.GeocoderComboBox', {
             });
 
             if (me.map) {
-                me.map.addLayer(me.locationLayer);
+                me.setMap(me.map);
```

Comment:
No need for this, should be done automatically by ExtJS mechanism by calling `callParent` below.